dowhy/causal_estimators/linear_regression_estimator.py

```diff
@@ -37,11 +37,11 @@ def _build_model(self):
     def _estimate_confidence_intervals(self, confidence_level,
             method=None):
         conf_ints = self.model.conf_int(alpha=1-confidence_level)
-        return conf_ints.to_numpy()[1:(len(self._treatment_name)+1),:]
+        return (self._treatment_value - self._control_value) * conf_ints.to_numpy()[1:(len(self._treatment_name)+1),:]
```

Comment:
can you write an inline comment on why the conf int is multiplied by this? same for the standard error below. It will be good to give a justification here for future readers of the code.

Reply:
I have added the comments. Could you look into it?